src/lib/Canvas/Canvas.svelte

```diff
@@ -58,10 +67,23 @@
         });
 	}
 
+    function get_color(l2id) {
+        let net = $networks[l2id];
+        // zla siet zla farba sme rasisti
+        if (net == undefined) {
+            return "#000000";
+        }
+        return $networks[l2id].color;
+    }
+
```

Comment:
túto funkciu netreba, či?

Reply:
nn to netreba, som zabudol vymazat, takto sa nedaju dynamicky updatovat farby

Reply:
dole máš suggestion, skús, že čo to spraví xd